### ISSUE TITLE
Consistent format and Readability for FAQs 

### ISSUE THREAD

**Issue by saralab** [hirosystems/docs#506]
We have valuable info on the FAQs that are way harder to find. Let's organize this content better; I needed help parsing through the massive wall of text to get to anything I wanted to find out. 

We should make it fast and easy for our Devs to parse through this info.

Few suggestions: 
- Enable expand-collapse for each question under FAQ: this will allow Devs to quickly scan for the problem they're trying to solve 
- Group the FAQs under neat categories 
- Where step-by-step guides are involved, number the bullets

This obviously needs TLC from design.

**Comment by ryanwaits** [comment]
we have some plans for this, but still needs some work/thought on the implementation.

tldr; a lot of this information can be put into one-off guides or content pieces that live inside of our new Guides section, but that section needs to be designed and thought out
